examples/login/login.py

```diff
@@ -1,61 +1,58 @@
+"""This example demonstrates how to log in to a website using AgentQL."""
+
 import time
-import agentql
 
-URL = "https://www.target.com"
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+URL = "https://www.yelp.com/"
 EMAIL = "REPLACE_WITH_YOUR_EMAIL (For target.com)"
 PASSWORD = "REPLACE_WITH_YOUR_PASSWORD (For target.com)"
 
-if __name__ == "__main__":
-
-    # Start a session with the specified URL
-    session = agentql.start_session(URL)
-
-    # Define the queries to interact with the page
-    sign_in_query = """
-    {
-        sign_in_btn
-    }
-    """
+# Define the queries to interact with the page
+LOG_IN_QUERY = """
+{
+    log_in_btn
+}
+"""
 
-    credentials_query = """
-    {
+CREDENTIALS_QUERY = """
+{
+    sign_in_form {
         email_input
         password_input
-        sign_in_with_password_btn
+        log_in_btn
     }
-    """
+}
+"""
 
-    skip_query = """
-    {
-        skip_btn
-    }
-    """
-
-    # Make API call(s) to AgentQL server to fetch the query
-    response = session.query(sign_in_query)
-
-    response.sign_in_btn.click(force=True)
 
-    response_sign_in = session.query(sign_in_query)
+def main():
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=False)
 
-    response_sign_in.sign_in_btn.click(force=True)
+        # Create a new AgentQL page instance in the browser for web interactions
+        page: Page = browser.new_page()
```

Comment:
@frankfeng98 does VSCode show a warning at this line for you? Your linter might not be set up correctly if not. Can you double check this, pls, and let's fix the linter warning here before landing (i.e. `# type: ignore`)

Reply:
I'm not seeing Linter warning on my side. I'm wondering what kind of warning you were seeing, could you share it? The reason I'm asking is that I thought pylint setting is determined by `.pylintrc` file, which comes with the repo and should be standard for every TF developers. If this is indeed related to pylint setting, we might consider editing the `.pylintrc` file so that everyone could see this warning.